goentri.com.fourthwall.json

```diff
@@ -8,104 +8,19 @@
   "description": "Allows user to easily set up domain using Entri",
   "syncPubKeyDomain": "goentri.com",
   "syncRedirectDomain": "api.goentri.com, goentri.com",
-  "variableDescription": "ip is the website server ip, zendeskverification is the text verification value for zendesk, sendgridHost is sendgrid dynamic host.",
+  "variableDescription": "ip is the website server ip, fourthwallHost is the fourthwall host name to be added",
   "sharedProviderName": true,
   "records": [
     {
       "type": "A",
-      "host": "@",
+      "host": "%fourthwallHost%",
```

Comment:
this variable shall be now changed with `@`. The host will be then controlled by the `host` parameter when applying the template (and effectively replace `@`).